src/modules/create_items.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../classes/author'
+require_relative '../classes/book'
+require_relative '../classes/db'
+require_relative '../classes/game'
+require_relative '../classes/genres'
+require_relative '../classes/label'
+require_relative '../classes/movie'
+require_relative '../classes/music'
+require_relative '../classes/source'
+require_relative 'show_items'
+require_relative 'private_methods'
+
+module CreateItems
+  include ShowItems
+  include PrivateMethods
+
+  def valid_year?(num)
+    return unless num.length != 4
+
+    puts 'Please enter a valid year'
+    year = gets.chomp
+    valid_year?(year)
+  end
+
+  def create_movie
+    source = create_source
+    puts "\nWhat is the publish date of the movie? e.g. 1967"
+    year = gets.chomp
+    valid_year?(year)
+    puts "\nIt's the movie silent?"
```

Comment:
- [ ] If I try to select 1 for yes, I get my movie as silent: false anyways, please take a look at this small bug.